keras/backend/cntk_backend.py

```diff
@@ -936,12 +936,24 @@ def pow(x, a):
 
 
 def clip(x, min_value, max_value):
-    if max_value is not None and max_value < min_value:
-        max_value = min_value
-    if max_value is None:
-        max_value = np.inf
+    """Element-wise value clipping.
+
+    # Arguments
+        x: Tensor or variable.
+        min_value: Python float, integer or tensor.
+        max_value: Python float, integer or tensor.
+
```

Comment:
The theano and cntk backend functions don't need docstrings. The tensorflow backend docstrings are used for the documentation.